src/service_client/assisted_service_api.py

```diff
@@ -324,6 +325,16 @@ def download_and_save_file(self, cluster_id: str, file_name: str, file_path: str
         with open(file_path, "wb") as _file:
             _file.write(response.data)
 
+    def download_manifests(self, cluster_id: str, dir_path: str) -> None:
+        log.info("Downloading manifests for cluster %s into %d", cluster_id, dir_path)
```

Comment:
`TypeError: %d format: a number is required, not str`

Reply:
Use f-strings instead 

```suggestion
        log.info(f"Downloading manifests for cluster {cluster_id} into {dir_path}")
```

Reply:
fixed